### PR TITLE
Handle logic of methods using sat_version instead of two variables.

### PR DESCRIPTION
**Pull request by kbidarkar**
a) Earlier we had both sat_versions and upstream variables to
   handle the flow of logic.
b) Now all the logic flow will be depending upon the sat_version
   variable.
c) This will reduce complexity and simplify things.

### CODE REVIEW

**Comment by kbidarkar** [comment]
I could only find those instances , which required update. If there are any other instances for which we need to remove the upstream check, will do so. :)

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1545,24 +1545,21 @@ def product_install(distribution, create_vm=False, certificate_url=None,
...
-                execute(setup_oscap, host=host)
-            if satellite_version not in ('6.0', '6.1'):
-                execute(oscap_content, host=host)
+        if satellite_version != '6.0':
+            execute(install_puppet_scap_client, host=host)
+        if satellite_version == '6.1':
+            execute(setup_oscap, host=host)
+        if satellite_version not in ('6.0', '6.1'):
+            execute(oscap_content, host=host)
         # ostree plugin is for Sat6.2+ and nightly (rhel7 only)
```
Comment (lpramuk) [inline automation_tools/__init__.py]: lines 1554 to 1559 either needs to be indented into one common ```if``` or yet better now all 3 ifs can merged with logic sat version != nightly
1. ```if satellite_version not in ('6.0', 'nightly'):``` (added or not nightly)
2. ```if satellite_version == '6.1':``` (stays same)
3. ```if satellite_version not in ('6.0', '6.1', 'nightly'):``` (added or not nightly)

**Comment by lpramuk** [comment]
Implements issue #422

File: automation_tools/__init__.py (hunk 1 above)
Comment (kbidarkar) [inline automation_tools/__init__.py]: I wanted to avoid an indent , so removed it but missed to add back nightly checks in the if.

**Comment by kbidarkar** [comment]
@lpramuk , if you were referring to ostree plugin it does work for nightly.

**Comment by sghai** [comment]
ACK.. Looks good.

**Comment by sghai** [comment]
@lpramuk : Please feel free to merge if you'r fine with changes..

**Pull request merged**